### ISSUE TITLE
Prefixed version of gnused?

### ISSUE THREAD

**Issue by vr8hub** [jetpack-io/devbox#1965]
### Package name

gnused_prefixed

### Software

Would it be possible to get a prefixed version of gnused, like there is for coreutils? I need access to both the system sed and gnused, and with my package manager to date gnused was installed as gsed. (I really expected prefixed to be the default for both; I was surprised by coreutils and had to uninstall it and install coreutils_prefixed, after I discovered the existence of the latter.)

**Comment by vr8hub** [comment]
As a workaround, I've renamed the "sed" link in the devbox profile/default/bin directory to gsed (had to use sudo), but I assume that's not going to survive an update to the package, etc.?

**Comment by Lagoja** [comment]
Hey @vr8hub, it's not perfect, but one way to wok around this without creating a new nix package or writing Nix code would be to put something like this in the `init_hook`: 

```
      "alias gsed=$(which sed)",
      "alias sed=/usr/bin/sed",
```

This will run everytime you start the shell, so it should survive a package update or refresh

**Comment by vr8hub** [comment]
Yes, thank you. I can figure out workarounds, the point is that we shouldn't have to. I've been using gnu sed on the Mac for over ten years using a variety of packaging managers, and this is the first time it hasn't been installed as gsed. That simply should be the default—nothing should override system utilities indiscriminately, and without warning or means to do something different.
